ocean_lib/ocean/test/test_util.py

```diff
@@ -84,9 +83,15 @@ def test_get_web3_connection_provider(monkeypatch):
     provider = util.get_web3_connection_provider("rinkeby")
     assert provider.endpoint_uri == "https://rinkeby.infura.io/v3/id1"
 
+    # polygon network name
+    assert "polygon" in util.SUPPORTED_NETWORK_NAMES
```

Comment:
can you please add messages to asserts? per #283

Reply:
Done. https://github.com/oceanprotocol/ocean.py/pull/308/commits/bf92cda2a55964c7a98aa3972bf8cffa40976405